style.css

```diff
@@ -0,0 +1,126 @@
+*{margin:0;padding:0;box-sizing: border-box;}
+
+section {
+    margin: 0 auto;
+}
+
+.top-section{
+    height: 92px;
+    background: #3C3A39;
+}
+.top-bar {
+    width: 375px;
+    display: flex;
+    padding-top: 16px;
+}
+.top-bar div{
+    margin: 4px;
+}
+
+.time {
+    width: 50%;
+        padding-left: 21px;
+}
+
+.main-menu{
+    display: flex;
+padding: 16px 0 12px 18px
+}
+
+.john{
+    color: #FFFFFF;
+    font-weight: 600;
+    width: 60%;
+}
+
+
+.hero {
+    height: 590px;
+    background: #1C1A19;
+
+}
+
+h1 {  
+    padding-top: 120px;
+    padding-left: 24px;
+    font-family: 'Crete Round', serif;
+    left: 24px;
+    font-style: normal;
+    font-size: 46px;
+    letter-spacing: 0.37px;
+    color: #FF6B00;
+    /* position: static; */
+    /* width: 327px; */
+    /* height: 120px; */
+    /* top: 120px; */
+    /* font-weight: 400; */
+    /* line-height: 64px; */
+    /* flex: none; */
+    /* order: 0; */
+    /* flex-grow: 0; */
+}
+
+h3{
+    /* position: static; */
+    /* width: 327px; */
+    /* height: 15px; */
```

Comment:
- [ ] Please don't leave unused code as a comment in your code base, this will make your code not clean and unprofessional, I would suggest you remove it.